tests/lib/rules/space-in-parens.js

```diff
@@ -165,6 +166,12 @@ ruleTester.run("space-in-parens", rule, {
             options: ["never"],
             errors: [{ messageId: "rejectedClosingSpace" }]
         },
+        {
+            code: "foo(bar()\n)\n",
```

Comment:
This seems inconsistent with [this test](https://github.com/eslint/eslint/pull/12746/files#diff-a04da4863cf712137d65a202b5534e69R177). Do you mind explaining why these two cases should be treated differently?